text/0035-wotsicide/0035-wotsicide.md

```diff
@@ -0,0 +1,387 @@
+            <summary>TreasuryOutput</summary>
+            <table>
+                <tr>
+                    <td><b>Name</b></td>
+                    <td><b>Type</b></td>
+                    <td><b>Description</b></td>
+                </tr>
+                <tr>
+                    <td>Output Type</td>
+                    <td>byte</td>
+                    <td>
+                        Set to <strong>value 2</strong> to denote an <i>TreasuryOutput</i>.
+                    </td>
+                </tr>
+                <tr>
+                    <td>Amount</td>
+                    <td>uint64</td>
+                    <td>The amount of funds residing in the treasury.</td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+</table>
+
+### Treasury Input
+
+The `TreasuryInput` is equivalent to a normal `UTXOInput` but instead of referencing a transaction, it references a milestone. This input can only be used within `TreasuryTransaction` payloads.
+
+Serialized form:
+
+<table>
+    <tr>
+        <td><b>Name</b></td>
+        <td><b>Type</b></td>
+        <td><b>Description</b></td>
+    </tr>
+    <tr>
+        <td>Input Type</td>
+        <td>byte</td>
+        <td>
+            Set to <strong>value 1</strong> to denote an <i>TreasuryInput</i>.
+        </td>
+    </tr>
+    <tr>
+        <td>Milestone Hash</td>
+        <td>Array<byte>[32]></td>
+        <td>The hash of the milestone which created the referenced <i>TreasuryOutput</i>.</td>
+    </tr>
+</table>
+
+### Treasury Output
+
+The `TreasuryOutput` is a special output type which represents the treasury of the network, respectively the non yet migrated funds. At any given moment in time, there is only one `TreasuryOutput`.
+
+Serialized form:
+
+<table>
+    <tr>
+        <td><b>Name<b></td>
+        <td><b>Type</b></td>
+        <td><b>Description</b></td>
+    </tr>
+    <tr>
+        <td>Output Type</td>
+        <td>byte</td>
+        <td>
+            Set to <strong>value 2</strong> to denote an <i>TreasuryInput</i>.
+        </td>
+    </tr>
+    <tr>
+        <td>Amount</td>
+        <td>uint64</td>
+        <td>The amount of funds residing in the treasury.</td>
+    </tr>
+</table>
+
+The `TreasuryOutput` can not be referenced or spent by transactions, it can only be referenced by receipts.
+
+The `TreasuryOutput` can be queried from the HTTP API and needs to be included within snapshots in order to keep the total supply intact.
+
+## Receipts
+
+Receipts allow for fast migration of funds from the legacy into the new network by representing entries of funds which were migrated in the old network.
+
+### Schema
+Receipts are listings of funds for which nodes must generate UTXOs in the form of `SigLockedSingleOutput`s targeting the given address. Receipts are embedded within milestone payloads and therefore signed by the Coordinator. A milestone may contain up to one receipt as a payload. The Coordinator chooses whether to embed a receipt payload or not.
```

Comment:
So currently there are no milestone payloads speced in the milestone RFC
Should we change it?

Or mention that we temporarily change ms structure here?